data_structures/HashMap/Hashing.dart

```diff
@@ -70,7 +70,8 @@ class HashMap {
   List<LinkedList> buckets;
 
   HashMap(int hsize) {
-    buckets = new List<LinkedList>(hsize);
```

Comment:
I think assigning length value is not a good practice. Can we change the constructor to List.filled()